src/wovnio/wovnphp/Store.php

```diff
@@ -260,4 +264,19 @@ public function compressApiRequests()
     {
         return $this->settings['compress_api_requests'];
     }
+
+    public function outboundProxy()
+    {
+        $result = null;
+
+        if (isset($this->settings['outbound_proxy_host'])) {
+            $result = $this->settings['outbound_proxy_host'];
+        }
+        if (isset($this->settings['outbound_proxy_port'])) {
+            $port = $this->settings['outbound_proxy_port'];
+            $result .= ":$port";
+        }
+
+        return $result;
```

Comment:
Why do we need to split host and port? How about having configuration to be connected.
e.g. `outbound_proxy_host_and_port: site.com:8000`

Reply:
Seems like there's a `CURLOPT_PROXYPORT`:
https://www.php.net/manual/en/function.curl-setopt.php (under "options")

If we want, we can use that to specify the port.

Reply:
to be consistent with wovnjava 😎

Reply:
They do mention this re CURLOPT_PROXYPORT
> The proxy's port number may optionally (but discouraged) be specified with the separate option [CURLOPT_PROXYPORT](https://curl.se/libcurl/c/CURLOPT_PROXYPORT.html).